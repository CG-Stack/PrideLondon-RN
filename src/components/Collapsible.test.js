// @flow
import React from "react";
import { Text } from "react-native";
import { shallow } from "enzyme";
import Collapsible from "./Collapsible";

it("renders correctly", () => {
  const output = shallow(
    <Collapsible>
      <Text>Hello</Text>
    </Collapsible>
  );

  expect(output).toMatchSnapshot();
});
