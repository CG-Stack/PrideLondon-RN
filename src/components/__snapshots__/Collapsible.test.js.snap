// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Component>
  <Component>
    <AnimatedComponent
      onLayout={[Function]}
      style={
        Object {
          "height": 0,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Hello
      </Text>
    </AnimatedComponent>
  </Component>
</Component>
`;
